packages/viur-vue-utils/bones/edit/default/textBone.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <template v-if="state.editor">
     <ckeditor
-      v-if="boneState.bonestructure['validHtml']"
+      v-if="boneState.bonestructure['valid_html']"
```

Comment:
If you use an old core this should still work IMO, so I would support both:
```suggestion
      v-if="boneState.bonestructure['valid_html'] ?? boneState.bonestructure['validHtml']"
```